pkg/utils/database/mysql.go

```diff
@@ -195,3 +196,33 @@ func (m Mysql) GetCredentials() Credentials {
 		Password: m.Password,
 	}
 }
+
+// ParseAdminCredentials parse admin username and password of mysql database from secret data
+// If "user" key is not defined, take "root" as admin user by default
+func (m Mysql) ParseAdminCredentials(data map[string][]byte) (AdminCredentials, error) {
```

Comment:
@hyunysmile what do you think to add the keys for password/username to the instance as properties, so it's dynamic in the future and we only fallback to "user" and "password", if not set.

(minor)

Reply:
we can think about it when there are more cases that we need to handle.